Assets/Scripts/Audio/AudioManager/MusicManager.cs

```diff
@@ -2,30 +2,8 @@
 using System.Collections;
 
 public class MusicManager : AbstractAudioManager {
-    
-    //Singleton Instantiation
-    public static MusicManager instance;
 
-    void Awake() {
-        if (instance != null) {
-            Debug.LogError("Multiple instances of SoundEffectsHelper!");
-        }
-        instance = this;
-    }
-
-    public static MusicManager getInstance() {
-        return (MusicManager) instance;
-    }
-
-    //Songs
-    public AudioSource mainTheme;
-    public AudioSource mechanicalTheme;
-
-    //Song Volume
-    public float mainThemeVolume;
-    public float mechanicalThemeVolume;
-
-    public float fadeInSpeed;
+	public float fadeInSpeed;
```

Comment:
I still think we should set fadeInSpeed dependent on each song. Because some songs its better to fade in quick while other songs slowly (for example the happy Puppet master song should slowly fade into the creepier version).